src/main/java/hill/weatherconfidence/ingest/model/GeocodeResponse.java

```diff
@@ -0,0 +1,16 @@
+package hill.weatherconfidence.ingest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.ArrayList;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GeocodeResponse extends ArrayList<GeocodeItem> {
```

Comment:
There's probably a way to get the web client to handle this list wrapper automatically using generics but I haven't looked into the docs yet to find out how to do that.  Meantime, a simple wrapper class does the job